frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeListPage/CodeLists/CodeLists.test.tsx

```diff
@@ -246,6 +246,8 @@ describe('CodeLists', () => {
 
   it('calls onDeleteCodeList when clicking delete button', async () => {
     const user = userEvent.setup();
+    jest.spyOn(window, 'confirm').mockImplementation(jest.fn(() => true));
```

Comment:
Småpirk: Er det nødvendig å pakke denne i `jest.fn()`?
```suggestion
    jest.spyOn(window, 'confirm').mockImplementation(() => true);
```